src/webhooks/models/webhooksFormatter.js

```diff
@@ -26,12 +26,13 @@ function unknownWebhookEventTypeError(badWebhookEventTypeValue) {
     return new Error(`Unrecognized webhook event: ${badWebhookEventTypeValue}, must be one of the following: ${WEBHOOK_EVENT_TYPES.join(', ')}`);
 }
 
-function getThresholdMessage(state, { isSlack, testName, benchmarkThreshold, lastScores, aggregatedReport, score }) {
+function getThresholdMessage(state, { emoji, testName, benchmarkThreshold, lastScores, aggregatedReport, score }) {
     let resultText = 'above';
-    let icon = isSlack ? slackEmojis.ROCKET : teamsEmojis.ROCKET;
+    let icon = null;
+    icon = emoji.ROCKET;
     if (state === WEBHOOK_EVENT_TYPE_BENCHMARK_FAILED) {
         resultText = 'below';
-        icon = isSlack ? slackEmojis.CRY : teamsEmojis.CRYING;
+        icon = emoji.CRY;
     }
     return `${icon} *Test ${testName} got a score of ${score.toFixed(1)}` +
         ` this is ${resultText} the threshold of ${benchmarkThreshold}. ${lastScores.length > 0 ? `last 3 scores are: ${lastScores.join()}` : 'no last score to show'}` +
```

Comment:
I think the `emoji` parameter should be passed as a third parameter of the function instead of being part of the stats dictionary.